Source/WebCore/loader/LinkLoader.cpp

```diff
@@ -112,6 +112,10 @@ void LinkLoader::loadLinksFromHeader(const String& headerValue, const URL& baseU
             continue;
 
         LinkLoadParameters params { relAttribute, url, header.as(), header.media(), header.mimeType(), header.crossOrigin(), header.imageSrcSet(), header.imageSizes(), header.nonce(), ReferrerPolicy::EmptyString };
+
+        if (!header.referrerPolicy().isEmpty())
+            params.referrerPolicy = parseReferrerPolicy(header.referrerPolicy(), ReferrerPolicySource::ReferrerPolicyAttribute).value_or(ReferrerPolicy::EmptyString);
```

Comment:
Can we parse this earlier and and pass it into the `LinkLoadParameters` constructor?